src/main/java/org/prebid/server/auction/ImpMediaTypeConverter.java

```diff
@@ -0,0 +1,46 @@
+package org.prebid.server.auction;
+
+import com.iab.openrtb.request.Imp;
+import com.iab.openrtb.request.Video;
+import org.prebid.server.exception.PreBidException;
+import org.prebid.server.proto.openrtb.ext.request.ImpMediaType;
+import org.prebid.server.proto.openrtb.ext.response.BidType;
+
+import java.util.List;
+import java.util.Objects;
+
+public class ImpMediaTypeConverter {
```

Comment:
Better resolver, as we are working not with something like dto, but with different arguments